client/src/Booking.jsx

```diff
@@ -89,46 +97,64 @@ class Booking extends React.Component {
     });
   }
 
+  //invoked when user clicks eligible checkout date
+  bookingTotal() {
+    console.log('inside bookingTotal');
+    // return axios.get('http://localhost:3002/booking/bookingTotal', { params: {
+    //   campId: 0,
+    //   check_in_date: this.state.check_in_date,
+    //   check_out_date: this.state.check_in_date
+    //   }
+    // })
+    //testing version:
+    //make sure timestamps r coming in as dates I can parse
+    let inTime = moment().format();
+    let outTime = moment().format();
+    return axios.get('http://localhost:3002/booking/bookingTotal', { params: {
+      campId: 0,
+      check_in_date: inTime,
+      check_out_date: outTime
+      }
+    })
+      .then((res) => {
+        console.log('res: ', res);
+        // this.setState({
+        // });
+      })
```

Comment:
Did this work? If so, are you planning to get rid of this but if code or refactor it?

Reply:
Yes, it works! I am planning to refactor so this axios call is triggered by a user clicking an eligible check out date. this is just here for testing of the route.